### PR TITLE
WIP: Remove project id

### PR DESCRIPTION
**Pull request by fisherxu**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug

 /kind cleanup
> /kind design
> /kind documentation
> /kind test
> /kind failing-test
> /kind feature

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: build/edge/kubernetes/03-configmap-edgenodeconf.yaml [hunk 1]
```diff
@@ -17,7 +17,7 @@ data:
         session-queue-size: 100 # A size of how many sessions will be handled. default to 100.
     edgehub:
         websocket:
-            url: wss://0.0.0.0:10000/e632aba927ea4ac2b575ec1603d56f10/edgenode1/events
+            url: wss://0.0.0.0:10000/edgenode1/events
```
Comment (kadisi) [inline build/edge/kubernetes/03-configmap-edgenodeconf.yaml]: It is recommended to modify `setup.md` to add instructions。

Comment (kadisi) [inline build/edge/kubernetes/03-configmap-edgenodeconf.yaml]: if projectid is useless, I recommend  formatting the `url` to `0.0.0.0:10000` .    then we can automatically assemble final url by `websocket.url` and `controller.node-id` or `edged.hostname-override`. But this needs to deal with compatibility issues。

In addition, `controller.node-id` and `edged.hostname-override` have duplicate settings problem.

Comment (fisherxu) [reply build/edge/kubernetes/03-configmap-edgenodeconf.yaml]: > In addition, controller.node-id and edged.hostname-override have duplicate settings problem.

Agreed, but nodeName will be used in Edgehub and Edged, so we also need to think where to place the nodeName field. And I think we can rename all the nodeid to nodeName which is more friendly to developer.

Comment (kadisi) [reply build/edge/kubernetes/03-configmap-edgenodeconf.yaml]: +1

**Pull request closed without merging**